content/_data/changelogs/weekly.yml

```
@@ -8863,6 +8863,60 @@
   # pull: 4927 (PR title: Added localizaton for devsecops=DevSecOps)
   # pull: 4930 (PR title: [JENKINS-63639] Update commons-io to 2.8.0)
 
+- version: '2.258'
+  date: 2020-09-22
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4942
+    issue: 62661
+    authors:
+    - res0nance
+    message: |-
+      Migrate the view status filter from views in previous Jenkins releases (regression in 2.240).
+  - type: rfe
+    category: rfe
+    pull: 4926
+    issue: 63592
+    authors:
+    - fqueiruga
+    message: |-
+      Revamp the empty state screens.
```

not sure if a user know's what this means? but also not sure on the best phrasing

---

How about:

> Improve the layout and clarity of the page displayed when jobs are not yet defined.

---

how about this, just changed the last word:
```suggestion
      Improve the layout and clarity of the page displayed when jobs are not yet created.
```